### ISSUE TITLE
Website - "Remember Me" option no longer working even when clicked when logging in

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3799]
**[Submitted to the original trac issue database at 3.43am, Thursday, 2nd June 2011]**

Just as the tile says, the "Remember Me" option is no longer working.  Every time I log into the site (on my home computer), I always click the "Remember Me" option so I don't have to re-login.  I have even deleted the OSM cookies (I just searched and manually deleted them so I didn't have my other logged in sites effected) and still can't get the "Remember Me" to work.

I considered adding this info to #3795, but it seems this *might* be a separate problem.

Anyways, this is happening with both IE-9 and FF 4.0.1.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 9.54am, Thursday, 2nd June 2011]**

Fixed and deployed.

**Comment by tomhughes** [comment]
**Author: rickmastfan67**
**[Added to the original trac issue at 3.50am, Wednesday, 16th November 2011]**

This problem is back again. :(

Happening in both IE-9 and FF 9.0 (Beta).

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.14am, Wednesday, 16th November 2011]**

Please don't reopen old bugs just because the symptoms are the same - this will be something entirely new related to the rails 3 upgrade.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 9.00am, Wednesday, 16th November 2011]**

I would also add that it seems to be working for me anyway - my session cookie has a one month expiry as expected.

**Comment by tomhughes** [comment]
**Author: rickmastfan67**
**[Added to the original trac issue at 9.36am, Wednesday, 16th November 2011]**

Well, whatever I do, I can't get it to stay logged in for me, even with the "Remember Me" box clicked.  I even deleted all my cookies for OSM in FF.  After I deleted the cookies, I logged in, closed FF and then went back to OSM, and I was logged out.  If you want me to open a totally new ticket for this still, let me know.
